doc/contribute/contributor_expectations.rst

```diff
@@ -87,27 +87,39 @@ Changes which require an RFC proposal include:
 Maintainers have the discretion to request that contributors create an RFC for
 PRs that are too large or complicated.
 
+.. _pr_requirements:
+
 PR Requirements
 ***************
 
 - Each commit in the PR must provide a commit message following the
   :ref:`commit-guidelines`.
 
+- No squash, fixup or merge commits are allowed, explicit or not.
```

Comment:
unsure on `squash` being here, continual updates to a PR can squash changes

Reply:
Yes, I think this is just a result of me thinking about the options in `git rebase -i`. I will remove it.